app/models/ruby_method.rb

```diff
@@ -23,12 +23,24 @@ def object_constant
     body[:object_constant]
   end
 
-  def instance_method?
-    method_type == "instance_method"
+  TYPES = {
+    "class" => {
+      identifier: "::"
+    }.freeze,
+    "instance" => {
+      identifier: "#"
+    }.freeze
+  }.freeze
+
+  TYPES.each do |type, _options|
```

Comment:
```suggestion
  TYPES.each_key do |type|
```

Reply:
Yes, thank you! RuboCop catches things like this, but there is Standard… fixed!